policies/coding-style.md

```diff
@@ -642,11 +642,74 @@ it portable enough to meet our multi-platform support goals.
 
 ## Chapter 14: Portability
 
+### Language
+
 To maximise portability the version of C defined in ISO/IEC 9899:1990
 should be used. This is more commonly referred to as C90. ISO/IEC 9899:1999
 (also known as C99) is not supported on some platforms that OpenSSL is
 used on and therefore should be avoided.
 
+### Integer types
+
+The types `int8_t`, `uint8_t`, `int16_t`, `uint16_t`, `int32_t`, `uint32_t`,
+`int64_t` and `uint64_t` are recommended for use when the exact width is
```

Comment:
what about `uint128_t`? It would be useful if only for handling result of `uint64_t` multiplication on 64 bit platforms...

Reply:
AFAIR `uint128_t` is hardly even standardized and not supported by all the compilers.

Reply:
gcc and clang support them, gcc has for something like 10 years now (`__uint128_t`)
That being said, MSVC only has the "return the high 64 bits intrinsic", no 128 bit type.

To be clear, I'm not saying that it should be always available, there is no point in it on 32 bit system for one, and there's little point when it can't be used for multiplication...

Reply:
> AFAIR `uint128_t` is hardly even standardized and not supported by all the compilers.

It's not standardized because ABI. Relevant [background](https://github.com/openssl/openssl/pull/15830#issuecomment-870474738)